### DISCUSSION TITLE
TypeError: fcst.predict(h=5)

### DISCUSSION THREAD

**Discussion by draconcio12** [Nixtla/mlforecast#263] [Q&A]
Trying to use predict I got the next error: 

--> 586 forecasts = ts.predict(
    587     models=self.models_,
    588     horizon=h,
    589     dynamic_dfs=dynamic_dfs,
    590     before_predict_callback=before_predict_callback,
    591     after_predict_callback=after_predict_callback,
    592     X_df=X_df,
    593     ids=ids,
    594 )
    595 if level is not None:
...
-> 1334         raise integer_op_not_supported(self)
   1335     obj = cast("PeriodArray", self)
   1336     result = obj._addsub_int_array_or_scalar(other * obj.freq.n, operator.add)

TypeError: Addition/subtraction of integers and integer-arrays with DatetimeArray is no longer supported.  Instead of adding/subtracting `n`, use `n * obj.freq`

**Comment by jmoralez** [comment]
Hey. This probably means that there's a mismatch between the frequency you declared in the MLForecast constructor and the actual frequency of your series. Can you please verify their formats?
